.github/workflows/nightly.yml

```diff
@@ -0,0 +1,42 @@
+name: Nightly tests
```

Comment:
A bit confusing, the term is usually used for something else, sounds like it's a beta workflow :)